prowler/lib/outputs/compliance/compliance.py

```diff
@@ -203,12 +204,14 @@ def get_check_compliance(finding, provider_type, output_options) -> dict:
     }
     """
     try:
+        if not isinstance(finding, FindingOutput):
+            check_id = finding.check_metadata.CheckID
+        else:
+            check_id = finding.check_id
```

Comment:
Why this change? The HTML should get this from the finding object.

Reply:
True